src/java/org/apache/cassandra/db/lifecycle/LogFile.java

```diff
@@ -504,7 +507,7 @@ List<String> getFilePaths()
 
     private String getFileName()
     {
-        return StringUtils.join(SSTableFormat.Type.current().info.getLatestVersion(), LogFile.SEP, // remove version and separator when downgrading to 4.x is becomes unsupported
+        return StringUtils.join(version, LogFile.SEP, // remove version and separator when downgrading to 4.x is becomes unsupported TODO should this include format type as well?
```

Comment:
> TODO should this include format type as well?

if the version does not include the format, I think it would be good... `nc` isn't useful as I don't know this is `big` or `bti`

Another option is to use `version.toFormatAndVersionString`?

Reply:
Yes, I think that I oversaw that in CEP-17